corehq/apps/locations/bulk_management.py

```diff
@@ -715,6 +720,27 @@ def _check_location_names(self):
                     )
         return errors
 
+    @memoized
+    def _check_model_validation(self):
+        errors = []
+        for location in self.locations:
```

Comment:
There shouldn't be. These are the locations from the sheet, and they are already in memory. The import code loops through them a few other locations as well.